modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -285,6 +285,17 @@ const EditorManager: EditorManager = {
     self.suffix = suffix;
 
     FocusController.setup(self);
+
+    // Lock certain properties to reduce misuse
+    Arr.each(
+      [ 'majorVersion', 'minorVersion', 'releaseDate', 'pageUid' ],
+      (property) =>
+        Object.defineProperty(self, property, {
+          writable: false,
+          configurable: false,
+          enumerable: false,
```

Comment:
Are we sure that we don't want this to be enumerable. The new `pageId` I get but not enumerate on the other ones that has been in there for years might maybe break things? Maybe I'm overly cautious.

Reply:
That's a fair point. I will look more into this

Reply:
I will just leave `enumerable` true for all the locked properties